minitwit/src/main/java/WebApplication.java

```diff
@@ -878,4 +943,105 @@ else if (request.requestMethod().equals("POST")) {
         map.put("latest", LATEST);
         return map;
     };
+
+    public static Route serveMetrics = (Request request, Response response) -> {
+        return metrics.metrics();
+    };
+
+    public static Route serveStats = (Request request, Response response) -> {
+        SQLite db = new SQLite();
```

Comment:
Once we switch to the new Postgres database, we have to remember to change this line.